integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-unhappy.test.ts

```diff
@@ -0,0 +1,49 @@
+import { HistoryEvent } from "@aws-sdk/client-sfn";
+import { SfnContainerHelper } from "./sfn-container-helper";
+
+jest.setTimeout(30_000);
+
+describe("nino-issue-credential-unhappy", () => {
+  let sfnContainer: SfnContainerHelper;
+
+  beforeAll(async () => {
+    sfnContainer = new SfnContainerHelper();
+  });
+
+  afterAll(async () => sfnContainer.shutDown());
+
+  it("has a step-function docker container running", async () => {
+    expect(sfnContainer.getContainer()).toBeDefined();
+  });
+
+  it("should fail when nino check is unsuccessful", async () => {
```

Comment:
`should fail when nino check is unsuccessful` this isn't check nino

- Failures in signing jwt can be as a result of the following see https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/4ee8b1e6132bfc9b705fb6280589e415d44936db/lambdas/jwt-signer/tests/jwt-signer-handler.test.ts#L98